main.py

```diff
@@ -20,6 +21,7 @@ def connect_to_amqp():
     yield amqp_con.declare_queue("mobile_production_queue")
     yield amqp_con.declare_queue("heroku_integration_queue")
     yield amqp_con.declare_queue("heroku_production_queue")
+    yield amqp_con.declare_queue("cloudtrail_production_queue")
```

Comment:
don't you need an integration version ?

Reply:
Since we have only a single AWS account for both environments, we don't need it.

Reply:
Actually, I think you should add an integration version even if we don't use it, for completeness :)